pkg/api/roles.go

```diff
@@ -244,10 +244,44 @@ func (hs *HTTPServer) declareFixedRoles() error {
 		Grants: []string{string(models.ROLE_ADMIN)},
 	}
 
+	annotationsWriterRole := accesscontrol.RoleRegistration{
```

Comment:
Fixed role assignments will be a bit tricky - seems like these are the current access restrictions:
* everyone can list annotations and annotation tags;
* organisation admins can mass delete annotations;
* organisation editors and admins can create Graphite annotations;
* organisation editors can create and update global annotations (where dashboardID is 0, not associated with any particular dashboard);
* users who have access to edit a dashboard can create and update annotations associated with this dashboard.

So listing annotations and tags is easy - we can just grant these permissions to everyone (as you've done).

However, deciding on whom to grant the rest of the permissions is more difficult, as we're proposing that endpoints that have different legacy access control restrictions will share the same FGAC permissions.

I'd suggest limiting this PR to covering only the `GET` endpoints. We'll need to give the rest of it some more thought.

Hope this makes sense!

Reply:
Thanks por pointing this out. I will split this PR so this one will only make changes on `GET` endpoints.